mcs/class/Mono.Debugger.Soft/Mono.Debugger.Soft/VirtualMachine.cs

```diff
@@ -277,6 +277,13 @@ public void SetSocketTimeouts (int send_timeout, int receive_timeout, int keepal
 			return new ExceptionEventRequest (this, exc_type, caught, uncaught);
 		}
 
+		public ExceptionEventRequest CreateExceptionRequest2 (TypeMirror exc_type, bool caught, bool uncaught, bool everything_else) {
+			if (Version.AtLeast (2, 54))
```

Comment:
This could have the name same since it has a different signature.

Reply:
Done